docs/configuration.md

```diff
@@ -0,0 +1,124 @@
+is a setting which will allow servers to delete themselves after a period of inactivity.
+
+### Frequency
+By default the culler runs every 10 minutes. With a more aggressive setting for the notebook
+idle timeout the hub-managed culler can be run less frequently.
+
+### Concurrency limit
+By default the culler has a concurrency limit of 10. This means it will make up to 10
+concurrent API calls. When deleting a large number of users that can generate a high load
+on the hub. Setting this to `1` helps to reduce load on the hub.
+
+### Timeout
+The timeout controls how long a server can be idle before being deleted. Because the servers
+will aggressively cull themselves this value can be set very high.
+
+These can be all configured in the `cull` section of `values.yaml`:
+```yaml
+cull:
+  timeout: 432000 # 5 days
+  every: 3600 # Run once an hour instead of every 10 minutes
+  concurrency: 1
+```
+
+### Notebook culler
+How long a server can be idle before culling itself is controlled by the `IDLE_TIMEOUT`
+environment variable. This is configured in `singleuser.extraEnv` in `values.yaml`.
+
+```yaml
+singleuser:
+  extraEnv:
+    IDLE_TIMEOUT: 14400 # 4 hours
+```
+
+## Activity intervals
+These settings control how spawner and user activity is tracked. These settings have
+a large impact on the performance of the hub.
+
+### `c.JupyterHub.activity_resolution`
+Activity resolution controls how often activity updates are written to the database. Many
+API calls will record activity for a user. This setting determines whether or not that update
+is written to the database. If the update is more recent than `activity_resolution` seconds
+ago it's ignored. Increasing this value will reduce commits to the database.
+
+```yaml
+extraConfig:
+  myConfig: |
+    c.JupyterHub.activity_resolution = 6000
+```
+
+### `c.JupyterHub.last_activity_interval`
+This setting controls how often a periodic task in the hub named `update_last_activity`
+runs. This task updates user activity using information from the proxy. This task makes
+a large number of database calls and can put a fairly significant load on the hub. Zero to
+Jupyterhub sets this to 1 minute by default. The upstream default of 5 minutes is a better
+setting.
+
+```yaml
+extraConfig:
+  myConfig: |
+    c.JupyterHub.last_activity_interval = 300
+```
+
+### `JUPYTERHUB_ACTIVITY_INTERVAL`
+This controls how often each server reports its activity back to the hub. The default
+is 5 minutes and with hundreds or thousands of users posting activity updates it puts
+a heavy load on the hub and the hub's database. Increasing this to one hour or more
+reduces the load placed on the hub by these activity updates.
+
+```yaml
+singleuser:
+  extraEnv:
+    JUPYTERHUB_ACTIVITY_INTERVAL: "3600"
+```
+
+## Startup time
+
+### `init_spanwners_timeout`
+`c.JupyterHub.init_spawners_timeout` controls how long the hub will wait for spawners to
```

Comment:
```suggestion
[c.JupyterHub.init_spawners_timeout](https://jupyterhub.readthedocs.io/en/stable/api/app.html#jupyterhub.app.JupyterHub.init_spawners_timeout) controls how long the hub will wait for spawners to
```

In general I like to link / refer to things as much as possible for further digging.